src/log.ts

```diff
@@ -98,6 +98,11 @@ export function debug(..._: any[]) {
 export namespace message {
   export const INVALID_SPEC = 'Invalid spec';
 
+  // FIT
+  export const FIT_NON_SINGLE = 'Autosize fit only works well for single views that are not composed of other views except if layered.';
+
+  export const FIT_DATA_DRIVEN = 'Autosize fit does not work with data driven width or height.';
```

Comment:
We never talk about "data driven" width/height before (except in the new part of docs you added) so this is unclear that this means using range step. 

A better warning would be `'Cannot use "rangeStep" when "autosize" is "fit"'`.

Reply:
Done